packages/builder-core/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "builder-core",
+  "version": "0.0.1",
+  "author": "SEE package.json from PROJECT ROOT",
+  "contributors": "SEE package.json from PROJECT ROOT",
+  "license": "Apache-2.0",
+  "repository": "SEE package.json from PROJECT ROOT",
+  "main": "build/lib/lib.js",
+  "types": "build/lib/lib.d.ts",
+  "description": "",
+  "keywords": [
+    "smart-contracts",
+    "buidler",
+    "blockchain",
+    "dapps",
+    "javascript",
+    "tooling",
+    "task-runner"
+  ],
+  "bin": {
+    "buidler": "build/cli/cli.js"
+  },
+  "engines": {
+    "node": ">=8.2.0"
+  },
+  "scripts": {
+    "lint:fix": "npm run lint -- --fix",
+    "lint": "npm run lint",
+    "test": "node ../../node_modules/mocha/bin/mocha",
+    "coverage": "node ../../node_modules/nyc/bin/nyc.js ../../node_modules/mocha/bin/mocha",
+    "build": "node ../../node_modules/typescript/bin/tsc --build .",
+    "clean": "node ../../node_modules/rimraf/bin.js builtin-tasks internal *.d.ts *.map *.js tsconfig.tsbuildinfo build"
+  },
+  "devDependencies": {
+    "@types/chai": "^4.2.0",
+    "@types/debug": "^4.1.4",
+    "@types/download": "^6.2.4",
+    "@types/semver": "^6.0.2",
+    "chai": "^4.2.0",
+    "sinon": "^9.0.0",
+    "time-require": "^0.1.2"
+  },
+  "dependencies": {
+    "ansi-escapes": "^4.3.0",
+    "chalk": "^2.4.2",
+    "cross-env": "5.0.5",
+    "fs-extra": "^7.0.1",
+    "glob": "^7.1.3",
+    "raw-body": "^2.4.1",
+    "semver": "^6.3.0",
+    "source-map-support": "^0.5.13",
+    "ts-essentials": "^2.0.7"
```

Comment:
I think 3 above dependencies should be in `devDependencies`

Reply:
```
  "devDependencies": {
    "@types/chai": "^4.2.0",
    "chai": "^4.2.0",
    "time-require": "^0.1.2",
    "cross-env": "5.0.5",
    "source-map-support": "^0.5.13"
  },
  "dependencies": {
  },
```